Sources/TensorFlow/Operators/Math.swift

```diff
@@ -2158,6 +2158,36 @@ internal func _vjpMatmul<Scalar: TensorFlowFloatingPoint>(
     })
 }
 
+public extension Tensor where Scalar: TensorFlowFloatingPoint {
+    /// Computes the QR decomposition of each inner matrix in `tensor` such that
+    /// `tensor[..., :, :] = q[..., :, :] * r[..., :,:])`
+    /// 
+    /// - Parameters:
+    ///   - fullMatrices: If true, compute full-sized `q` and `r`. If false
+    ///     (the default), compute only the leading `P` columns of `q`.
+    ///  
+    func qr(fullMatrices: Bool = false) -> (Tensor<Scalar>, Tensor<Scalar>) {
```

Comment:
Avoid abbreviations and use terms of art. See [Swift API Design Guidelines - Use Terminology Well](https://swift.org/documentation/api-design-guidelines/#use-terminology-well) for more details.
```suggestion
    func qrDecomposition(fullMatrices: Bool = false) -> (q: Tensor<Scalar>, r: Tensor<Scalar>) {
```

Reply:
Done.